wandb/sdk/artifacts/storage_policies/wandb_storage_policy.py

```diff
@@ -169,10 +172,11 @@ def load_reference(
         manifest_entry: "ArtifactManifestEntry",
         local: bool = False,
         dest_path: Optional[str] = None,
+        skip_cache: bool = False,
```

Comment:
i don't think you need this. you should be able to use `manifest_entry.skip_cache`

Reply:
I think I'll have to either
- leave this as-is, or 
- put in a different workaround/fix if I can

`manifest_entry` is already instantiated at this point, and it's not obvious whether setting `manifest_entry.skip_cache = True` -- even temporarily -- could have weird side effects (e.g. if not thread-safe)

also considered creating temporary deepcopy of the `manifest_entry` (e.g. `copied_entry = deepcopy(manifest_entry); copied_entry.skip_cache = True`), but this could get incredibly slow over a lot of entries -- so I'd rather avoid it if at all possible

Reply:
oops sorry my bad. i remember setting manifest_entry.skip_cache while adding files and didn't realize its not set while downloading files. you're right. you could leave it as is i think